Lib/test/test_winconsoleio.py

```diff
@@ -92,9 +92,11 @@ def test_open_name(self):
         f.close()
         f.close()
 
-        f = open('C:/con', 'rb', buffering=0)
-        self.assertIsInstance(f, ConIO)
-        f.close()
+        # Windows 11 changed MS-DOS device name handling
```

Comment:
Can you add a reference to the bpo here and in the other modified test?

```suggestion
        # bpo-45354: Windows 11 changed MS-DOS device name handling
```

Reply:
Done in latest commit